templates/module.rs.template

```diff
@@ -1,13 +1,13 @@
 use odra::prelude::*;
-use odra::Variable;
+use odra::Var;
 
-/// A module definition. Each module struct consists Variables and Mappings
+/// A module definition. Each module struct consists Vars and Mappings
```

Comment:
```suggestion
/// A module definition. Each module struct consists Vars and Mappings.
```

Reply:
There is a second part of this sentence line below